lua/tardis/screens/cl_screen_music.lua

```diff
@@ -11,33 +44,47 @@ TARDIS:AddScreen("Music", {id="music", menu=false, order=10}, function(self,ext,
 	label:SetText("Enter song URL (clientside only currently)")
 	label:DoLayout()
 
-	local text = vgui.Create( "DTextEntry3D2D", frame )
-	text.is3D2D = screen.is3D2D
-	text.sub3D2D = label:GetText()
-	text:SetFont(TARDIS:GetScreenFont(screen, "Default"))
-	text:SetSize( frame:GetWide()*0.5, frame:GetTall()*0.1 )
-	text:SetPos(frame:GetWide()*0.5 - text:GetWide()*0.5,frame:GetTall()*0.5 - text:GetTall()*0.5)
-	text.OnEnter = function()
-		ext:PlayMusic(text:GetValue())
+	local text_bar = vgui.Create( "DTextEntry3D2D", frame )
+	text_bar.is3D2D = screen.is3D2D
+	text_bar.sub3D2D = label:GetText()
+	text_bar:SetFont(TARDIS:GetScreenFont(screen, "Default"))
+	text_bar:SetSize( frame:GetWide()*0.5, frame:GetTall()*0.1 )
+	text_bar:SetPos(frame:GetWide()*0.5 - text_bar:GetWide()*0.5,frame:GetTall()*0.5 - text_bar:GetTall()*0.5)
+	text_bar.OnEnter = function()
+		ext:PlayMusic(text_bar:GetValue())
+	end
+
+	local playbutt=vgui.Create("DButton",frame)
+	playbutt:SetSize( frame:GetWide()*0.2, text_bar:GetTall())
```

Comment:
nit: formatting on this line
```suggestion
	playbutt:SetSize(frame:GetWide()*0.2, text_bar:GetTall())
```